packages/pointer_interceptor/example/integration_test/widget_test.dart

```diff
@@ -76,6 +76,11 @@ void main() {
       app.main();
       await tester.pumpAndSettle();
 
+      if (!_newSemanticsAvailable()) {
+        print('Skipping test: Needs flutter > 2.10');
+        return;
```

Comment:
Could we do these via `skip`, or do you actually have to run `main` and `pumpAndSettle` to be able to do the check?

Not a big deal either way since it's temporary code.

Reply:
Yeah, I tried to find a way to do a proper `skip`, but I couldn't call skip, and the check that I do is purely runtime (it needs to look at the html code of a rendered app). If we could do: `skip: !isMaster`, I'd do that :P